.ci/test-install-docs.sh

```diff
@@ -11,18 +11,48 @@ set -e
 export GOPATH=${GOPATH:-${HOME}/go}
 
 typeset -r docker_image="busybox"
+typeset -r kata_project_url="github.com/kata-containers"
+typeset -r test_repo="${kata_project_url}/tests"
+typeset -r test_repo_url="https://${test_repo}"
+typeset -r test_repo_dir="${GOPATH}/src/${test_repo}"
+typeset -r kata_project_dir="${GOPATH}/src/${kata_project_url}"
+
+typeset -r mgr="${test_repo_dir}/cmd/kata-manager/kata-manager.sh"
+
+die()
+{
+	local msg="$*"
+	echo >&2 "ERROR: $msg"
+	exit 1
+}
 
 info()
 {
 	local msg="$*"
 	echo "INFO: $msg"
 }
 
+# Grab a copy of the tests repository
+get_tests_repo()
+{
+	[ -d "${test_repo_dir}" ] && return
+
+	mkdir -p "${kata_project_dir}"
+
+	pushd "${kata_project_dir}"
+	git clone "${test_repo_url}"
+	popd
```

Comment:
just fyi :) you could also do `git clone "${test_repo_url}" "${kata_project_dir}"`

Reply:
Sure. Trying to keep the code as simple as possible.

Reply:
Oh - sorry, yes - I'll do that...